### ISSUE TITLE
Lots of noop frames

### ISSUE THREAD

**Issue by lbertge** [minerllabs/minerl#134]
In `MineRLTreechop-v0/r2g1tempting_capers_shapeshifter-14_12243-16213`, there are a ton of frames where the player is clearly moving, but the action vector corresponding to those frames are zero'ed (i.e. a noop). Camera angles seem to be OK (at least, they're nonzero).

My digging suggests that, instead of individual frames being dropped (and somehow replaced with the noop), entire swaths of consecutive frames have this problem. In this particular example I've found a chunk of about 1000 frames (starting at frame 1829) with the data showing no action. 

I don't really know the extent of this problem. I wrote some code to look at other Treechop datasets, to see if there were sequences of noop for longer than 200 frames (10 seconds):
```py
import glob
import os
import numpy as np

path = os.path.expanduser("~/data/MineRLTreechop-v0/")
rendereds = glob.glob(path+'*/rendered.npz')

moves = [
'action_attack',
'action_back',
'action_forward', 
'action_jump',
'action_left', 
'action_right', 
'action_sneak', 
'action_sprint'
]

def noop(data, idx):
    return not np.any([data[move][idx] for move in moves])

files = 0
for f in rendereds:
    fdata = np.load(f)
    count = 0
    for frame in range(len(fdata['action_attack'])):
        if noop(fdata, frame):
            count += 1
        else:            
            count = 0
        
        if count >= 200:
            print(f)
            files += 1
            break
```
I've found 37 files with such a property. Some datasets are probably correct, but the prevalence of noops makes me think that `r2g1tempting_capers_shapeshifter-14_12243-16213` is not the only incorrect one.

**Comment by marioyc** [comment]
Definitely not all of these 37 files are invalid, I checked on `r2g1excellent_pluot_behemoth-21_311-3838` which actually completes the task but makes pauses twice during the recording. @brandonhoughton I'm guessing you have access to the actual position of the players which could discard cases like this one

**Comment by brandonhoughton** [comment]
Thanks for this - we found that stream was invalid and have removed it in the next version of the dataset!

**Comment by lbertge** [comment]
Still not working. Looks like filename merely got changed to `v1_tempting_capers_shapeshifter-14_12243-16095`. Same issue with noop frames, and same issue warping at frame 2227 onwards (described by #130). Video looks exactly the same.

I've cross-referenced this problem with your viewer, and with my own scripts. I'm pretty sure this bug is still here.

**Comment by MadcowD** [comment]
@Ibertge We're going to blacklist this file in our data loader! The issue will be closed and when minerl 0.2.3 is out you should be able to use this.
